app/controllers/api/v1/targets_controller.rb

```diff
@@ -1,6 +1,10 @@
 module Api
   module V1
     class TargetsController < Api::V1::ApiController
+      def index
+        @targets = Target.all
+      end
+
       def create
         @target = current_user.targets.create!(target_params)
       rescue ActiveRecord::RecordInvalid => e
```

Comment:
It looks like this 'e' is the one reek doesn't like, maybe you should check & fix that in the branch you made for the 'create' endpoint